app/frontend/src/components/composer/Composer.js

```diff
@@ -79,7 +81,7 @@ function Composer() {
         tmp.draft = draft;
         tmp.createdBy = user.sub;
         setRecipe((prevRecipe) => ({...prevRecipe, draft: draft, createdBy: user.sub}));
-        console.log('Saving Recipe:', tmp);
+        //console.log('Saving Recipe:', tmp);
```

Comment:
Are we keeping logs as comments or deleting them completely if not needed anymore?

Reply:
It does not really make a difference, I will delete this one